ReactiveXComponent/Configuration/XCApiTags.cs

```diff
@@ -29,5 +29,13 @@ public static class XCApiTags
         public const string Json = "Json";
         public const string Bson = "Bson";
         public const string GzipJson = "GzipJson";
+        public const string WebsocketType = "type";
+        public const string VirtualHost = "virtualHost";
+        public const string BusSslEnabled = "sslEnabled";
+        public const string BusSslServerName = "sslServerName";
+        public const string BusSslCertPath = "sslCertPath";
```

Comment:
Certificate instead of Cert

Reply:
I can't change this as it's the value already existing in the xcApi file.